packages/lumx-react/src/components/image-block/ImageBlock.tsx

```diff
@@ -111,68 +111,54 @@ const ImageBlock: React.FC<ImageBlockProps> = ({
     ...props
-}: ImageBlockProps): ReactElement => {
-    const { onClick = null, ...restProps } = props;
-
-    return (
-        <div
-            className={classNames(
-                className,
-                handleBasicClasses({
-                    align,
-                    aspectRatio,
-                    captionPosition,
-                    prefix: CLASSNAME,
-                    size,
-                    theme,
-                }),
-                {
-                    [`${CLASSNAME}--fill-height`]: fillHeight,
-                    [`${CLASSNAME}--format-crop`]: aspectRatio && aspectRatio !== AspectRatio.original,
-                    [`${CLASSNAME}--format-original`]: !aspectRatio || aspectRatio === AspectRatio.original,
-                },
-            )}
-            {...restProps}
-        >
-            <Thumbnail
-                align={align}
-                className={`${CLASSNAME}__image`}
-                aspectRatio={aspectRatio}
-                size={size}
-                fillHeight={fillHeight}
-                image={image}
-                onClick={onClick}
-                theme={theme}
-            />
-            {(title || description || tags) && (
-                <div className={`${CLASSNAME}__wrapper`} style={captionStyle}>
-                    {(title || description) && (
-                        <div className={`${CLASSNAME}__caption`}>
-                            {title && <span className={`${CLASSNAME}__title`}>{title}</span>}
-                            {/* Add an `&nbsp;` when there is description and title. */}
-                            {title && description && '\u00A0'}
-                            {isObject(description) && description.__html ? (
-                                <span dangerouslySetInnerHTML={description} className={`${CLASSNAME}__description`} />
-                            ) : (
-                                <span className={`${CLASSNAME}__description`}>{description}</span>
-                            )}
-                        </div>
-                    )}
-                    {tags && <div className={`${CLASSNAME}__tags`}>{tags}</div>}
-                </div>
-            )}
-            {actions && <div className={`${CLASSNAME}__actions`}>{actions}</div>}
-        </div>
-    );
-};
+}: ImageBlockProps): ReactElement => (
+    <div
+        className={classNames(
+            className,
+            handleBasicClasses({
+                align,
+                captionPosition,
+                prefix: CLASSNAME,
+                size,
+                theme,
+            }),
+            {
+                [`${CLASSNAME}--fill-height`]: fillHeight,
+                [`${CLASSNAME}--format-crop`]: aspectRatio && aspectRatio !== AspectRatio.original,
+                [`${CLASSNAME}--format-original`]: !aspectRatio || aspectRatio === AspectRatio.original,
+            },
+        )}
+        {...props}
+    >
+        {children ? children : null}
```

Comment:
I thought `undefined` was a valid ReactNode. Could this be replaced to just `{children}` without the ternary operator?

Reply:
Nice catch!